docs/developers/lsp-prereqs.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I think this page works better in the developers / LongShortPair(LSP) section. Wdyt?

Reply:
Yea, I agree that makes more sense. Updated to be before the deploy-an-lsp section.